### PR TITLE
fix(DB): Blackwing Technician spawn and addon

### PR DESCRIPTION
**Pull request by claudiodfc**
## Changes Proposed:
- Added spawns based on sniffs and updated addon for their emote.
- Cleared spawns.

## Issues Addressed:

- Closes not reported issue

## SOURCE:

Brotalnia's sniffs

## Tests Performed:

- Tested in-game for DB errors

## How to Test the Changes:

1. Apply SQL and check for errors
2. They should have the crafting emote playing

## Known Issues and TODO List:

None

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1643656464415474000.sql [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+(84200, 13996, 469, 2677, 2677, 1, -7486.45, -1025.62, 408.641, 15260, 64),
+(84201, 13996, 469, 2677, 2677, 1, -7490.95, -1014.89, 408.643, 15260, 64),
+(84202, 13996, 469, 2677, 2677, 1, -7515.74, -960.099, 449.325, 15260, 64),
+(84203, 13996, 469, 2677, 2677, 1, -7518.07, -933.761, 457.659, 15260, 64);
+
+-- Emote crafting for Technicians
+UPDATE `creature_template_addon` SET `emote`=133 WHERE `entry`=13996;
+
+-- Clean-up
+DELETE FROM `linked_respawn` WHERE `guid` >= 84533 AND `guid` <= 85779;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1643656464415474000.sql:81]: use BETWEEN instead

File: data/sql/updates/pending_db_world/rev_1643656464415474000.sql [hunk 2]
```diff
@@ -0,0 +1,82 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1643656464415474000');
+
+-- All Blackwing Technician's spawns
+DELETE FROM `creature` WHERE `id1` IN (13996);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1643656464415474000.sql:4]: Delete GUID as well. That is the Primary key :D

**Comment by Nefertumm** [comment]
Excellent Claudio! Thanks for those spawns. Please, if you can, match the data with this one, so it doesn't break the whole script
https://github.com/azerothcore/azerothcore-wotlk/pull/10433

File: data/sql/updates/pending_db_world/rev_1643656464415474000.sql [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+(84201, 13996, 469, 2677, 2677, 1, -7490.95, -1014.89, 408.643, 15260, 64),
+(84202, 13996, 469, 2677, 2677, 1, -7515.74, -960.099, 449.325, 15260, 64),
+(84203, 13996, 469, 2677, 2677, 1, -7518.07, -933.761, 457.659, 15260, 64);
+
+-- Emote crafting for Technicians
+UPDATE `creature_template_addon` SET `emote`=133 WHERE `entry`=13996;
+
+-- Clean-up
+DELETE FROM `linked_respawn` WHERE `guid` >= 84533 AND `guid` <= 85779;
+DELETE FROM `smart_scripts` WHERE (`source_type` = 0 AND (`entryorguid` IN(-84616, -84615, -84614, -84606, -84605, -84603)));
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1643656464415474000.sql:82]: Do not delete those scripts.

**Comment by claudiodfc** [comment]
Oh snap I didn't see, I'll take care of it later sorry

File: data/sql/updates/pending_db_world/rev_1643656464415474000.sql (hunk 2 above)
Comment (claudiodfc) [reply data/sql/updates/pending_db_world/rev_1643656464415474000.sql:4]: Oh man, that's gonna be a nasty delete. So many Guids

File: data/sql/updates/pending_db_world/rev_1643656464415474000.sql (hunk 1 above)
Comment (claudiodfc) [reply data/sql/updates/pending_db_world/rev_1643656464415474000.sql:81]: Can't avoid the C mentality, keep forgetting these exist

File: data/sql/updates/pending_db_world/rev_1643656464415474000.sql (hunk 2 above)
Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1643656464415474000.sql:4]: Yea, but mandatory as GUID is the key field

**Comment by Lynethris** [comment]
Tested and work : 

WoWScrnShot_022722_145343

And don't break the PR 10433

File: data/sql/updates/pending_db_world/rev_1643656464415474000.sql [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+(84200, 13996, 469, 2677, 2677, 1, -7486.45, -1025.62, 408.641, 15260, 64),
+(84201, 13996, 469, 2677, 2677, 1, -7490.95, -1014.89, 408.643, 15260, 64),
+(84202, 13996, 469, 2677, 2677, 1, -7515.74, -960.099, 449.325, 15260, 64),
+(84203, 13996, 469, 2677, 2677, 1, -7518.07, -933.761, 457.659, 15260, 64);
+
+-- Emote crafting for Technicians
+UPDATE `creature_template_addon` SET `emote`= 133 WHERE `entry`=13996;
+
+-- Clean-up
+DELETE FROM `linked_respawn` WHERE `guid` BETWEEN 84533 AND 85779;
```
Comment (Nefertumm) [inline data/sql/updates/pending_db_world/rev_1643656464415474000.sql:81]: Why? This is needed for Vaelastrasz technicians

File: data/sql/updates/pending_db_world/rev_1643656464415474000.sql [hunk 5]
```diff
@@ -0,0 +1,78 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1643656464415474000');
+
+-- All Blackwing Technician's spawns
+DELETE FROM `creature` WHERE `id1` IN (13996) AND `guid` IN(84533, 84534, 84535, 84536, 84537, 84538, 84539, 84540, 84544, 84545, 84547, 84549, 84550, 84551, 84552, 84554, 84555, 84556, 84562, 84563, 84564, 84565, 84566, 84567, 84568, 84569, 84570, 84571, 84602, 84617, 84618, 84619, 84623, 84624, 84654, 84655, 84688, 84689, 84691, 84692, 84759, 84760, 84761, 84762, 84763, 85609, 85611, 85612, 85613, 85616, 85620, 85621, 85622, 85623, 85625, 85760, 85764, 85766, 85767, 85768, 85772, 85773, 85776, 85777, 85778, 85779);
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1643656464415474000.sql:4]: the deleted guids are not matching the guids that are being inserted

**Review by acidmanifesto — CHANGES_REQUESTED** [review]

**Comment by acidmanifesto** [comment]
Marked as WIP and Removed Request to be Reviewed and To Be Merged until the errors are corrected.

**Review by acidmanifesto — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1643656464415474000.sql (hunk 5 above)
Comment (Nefertumm) [reply data/sql/updates/pending_db_world/rev_1643656464415474000.sql:4]: Not yet resolved

**Pull request merged**